pkg/cmd/label/http.go

```diff
@@ -79,9 +80,10 @@ loop:
 	return labels, totalCount, nil
 }
 
-func min(a, b int) int {
-	if a < b {
-		return a
+func determinePageLimit(limit int) int {
+	// If limit is -1 we want to retrieve all labels
+	if limit < 0 || limit > defaultPageLimit {
```

Comment:
@samcoe this seems antithetical to your previous statement about throttling. Users will never be able to request more than 30 items which means more requests for repos with more than 30 labels. Why not allow a page size up to 100 which, IIRC, is enforced as the max in the service anyway? It's the chatty-vs-chunky decision.

Reply:
I am not sure I am following. I am confident the current logic is correct but the variable names I have used are making it confusing as to what is happening here. `defaultPageLimit` should be renamed `maxPageSize` and `limit` should be renamed `numRequestedItems` or similar. This function should return the `maxPageSize` is the user requests more than the `maxPageSize` or if the user as requested `-1`, otherwise return the `numRequestedItems`.

Reply:
Understood, though I must admit I missed that the `defaultPageLimit` was changed to 100 from 30. 😳